sql/sys_vars.cc

```diff
@@ -6008,7 +6008,7 @@ static bool check_locale(sys_var *self, THD *thd, set_var *var)
     {
       push_warning_printf(thd, Sql_condition::WARN_LEVEL_WARN, ER_UNKNOWN_ERROR,
                           "Can't process error message file for locale '%s'",
-                          locale->name);
+                          locale->name.str);
```

Comment:
(also applies to the change in `storage/mroonga/lib/mrn_multiple_column_key_codec.cpp`)
There was a wrong type warning. This should do it?

Reply:
yes. at some point in time `name` was `char*` later changed to `LEX_CSTRING` I suppose, and not all occasions were changed.

It's not only a compiler warning, with Visual Studio this actually prints garbage

Reply:
> yes. at some point in time `name` was `char*` later changed to `LEX_CSTRING` I suppose, and not all occasions were changed.

`git blame` fd247cc21f

Reply:
* #3538